src/data/accessibility-quiz.ts

```diff
@@ -529,7 +529,27 @@ const accessibilityQuiz = [
     Explanation:
       "The four broad guidelines of web accessibility are Perceivable, Operable, Understandable, Robust. They help ensure that web content is accessible to all users, including those with disabilities",
     Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
-  }
+  },
+  {
+    Question: "Which of the following is NOT the way to fix web accessibility errors?",
+    Answer: "Changing document color",
+    Distractor1: "Updating Low Contrast Text",
+    Distractor2: "Adding Missing Alternative Text for Images",
+    Distractor3: "Fix Empty Links",
+    Explanation:
+      "Fix Empty Links, Adding Missing Alternative Text for Images and Update Low Contrast Text are the common ways to fix the accessiblility error",
```

Comment:
This still needs to be fixed.
There is a lot of words in this sentence that don't need to be capitalized.
For example, "Fix Empty Links" should just be "Fixing empty links"
Same with "Adding Missing Alternative Text for Images" can just be "adding missing alternative text for images"

please refer to my previous review comment on the fixes for the capitalization 👍

Reply:
@AaadityaG 

Here is the previous review comment with the correct changes 

https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/791#discussion_r1355700533

hope that helps

Reply:
done! Can I hope (hacktoberfest) label?